src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/variadic_split.cpp

```diff
@@ -4,37 +4,36 @@
 
 #include <vector>
 
-#include "single_layer_tests/variadic_split.hpp"
+#include "single_op_tests/variadic_split.hpp"
 #include "common_test_utils/test_constants.hpp"
 
-using namespace LayerTestsDefinitions;
+using namespace ov::test;
```

Comment:
You don't need whole ov::test here
"using ov::test::VariadicSplitLayerTest" is enough

Reply:
Thanks, done!